### ISSUE TITLE
Storage modal non-responsive with items.

### ISSUE THREAD

**Issue by btfrise** [Rise-Vision/storage-client#298]
Steps to reproduce:
1. Add an image or video item to placeholder
2. Open in-app storage to select content for that item
3. Select item
4. Try to exit out of modal
Result:
Storage items cannot be selected. Exit (X) button at top right is not responsive.

How it should work:
Storage items should be able to be selected through the in-app storage modal. Exit button should close storage modal.

Link to Presentation:
http://preview.risevision.com/Viewer.html?type=presentation&id=cf066a37-acfe-46a3-a825-6d681e657242

OS:
OS X 10.10
Ubuntu 14.04

Chrome Ver:
41.0.2272.76

**Comment by btfrise** [comment]
This issue has been resolved. @tejohnso @mfisher11 can you summarize the cause of the issue before I close? Thanks.

**Comment by tejohnso** [comment]
@btfrise unfortunately I don't recall what the issue was here or when/how it was resolved.
